LSpec/Config.lean

```diff
@@ -0,0 +1,3 @@
+-- TODO: Config data structure more complicated than this
+structure Config where
+  temp : Bool
```

Comment:
Do we need this? Configuring the behavior of an imported package should be done with a proper option via `set_option`, I believe. So this probably wouldn't be the way